pa_matchbatch.sh

```diff
@@ -279,7 +315,13 @@ for currentItem in ${repoRevMap}; do
 		jiraProjectId=$(getJiraProjectIdFor "${public_name}")
 		jiraFixedInProject=$(getJiraFixedInFor "${public_name}")
 
-		versionsUsed="${versionsUsed}\n${public_name}:${fix_ver}"
+        if [[ $public_name = "puppet-runtime" ]]; then
```

Comment:
this block needs to be indented to match the rest of the pushd/popd block.

Reply:
This is weird, it seems to be indented on my local copy, will fix this.